clippy_lints/src/operators/mod.rs

```diff
@@ -943,6 +983,7 @@ impl<'tcx> LateLintPass<'tcx> for Operators {
                     rhs,
                     self.modulo_arithmetic_allow_comparison_to_zero,
                 );
+                manual_is_multiple_of::check(cx, e, op.node, lhs, rhs, self.min_and_mask_size, self.msrv);
```

Comment:
We probably want to move it into `e.span.from_expansion()`, I'm not sure why there are so many lints operating outside of that range, if Clippy usually doesn't want to work in external macros.

Reply:
Yes.

Reply:
I've also added a test to ensure we don't lint in external macros.